calendar-bundle/src/Resources/contao/models/CalendarModel.php

```diff
@@ -82,6 +82,11 @@
  */
 class CalendarModel extends Model
 {
+	public const BREADCRUMB_MODE_EXTEND = 'extend';
+
+	public const BREADCRUMB_MODE_OVERRIDE = 'override';
+
+	public const BREADCRUMB_MODES = [self::BREADCRUMB_MODE_OVERRIDE, self::BREADCRUMB_MODE_EXTEND];
```

Comment:
Did you add this constant for the `options` key in the DCA only?

Reply:
Personal preference. Obsolute now anyway, because I removed setting as discuessed.